### PR TITLE
feat: handle login function

### PR DESCRIPTION
**Pull request by martjellino**
Add feat: handle login function

### CODE REVIEW

File: src/components/Auth/components/Login.jsx [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+    }
+    
+    const handleSubmitLogin = async () => {
+        const { email, password } = loginData
+        const res = await fetch(`https://eventmakers-api.vercel.app/api/auth/login`, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({ identity: { email }, password }),
```
Comment (indrazm) [inline src/components/Auth/components/Login.jsx]: Ini akan error, karena harusnya langsung

```js
body: JSON.stringify({ email, password })
```

**Review by indrazm — CHANGES_REQUESTED** [review]
Nice and Clean. 
Only 1 thing.
Please test and change. :)

**Comment by martjellino** [comment]
> Nice and Clean. Only 1 thing. Please test and change. :)

I already have updated the code. Kindly please check again :)

**Review by indrazm — APPROVED** [review]
I'll Approve this. 
But the line 28, and 29 might be wrong, since the backend different. 
We will fix it later.

**Pull request merged**